mrp_brewing/models/stock.py

```diff
@@ -27,6 +32,17 @@ def get_on_hand(self):
         string="Is Internal",
         compute="_compute_is_internal")
 
+    @api.multi
+    @api.depends('quant_ids')
+    def _compute_lot_numbers(self):
+
+        for stock_move in self:
+            lot_numbers = stock_move.lot_numbers.split('/') if stock_move.lot_numbers else []

```

Comment:
no needs to initialize it before. just put a empty list

Reply:
done